homeassistant/components/tuya/sensor.py

```diff
@@ -138,6 +138,21 @@
         ),
         *BATTERY_SENSORS,
     ),
+    # Irrigator
+    # https://developer.tuya.com/en/docs/iot/categoryggq?id=Kaiuz1qib7z0k
+    "ggq": (
+        SensorEntityDescription(
+            key=DPCode.COUNTDOWN,
+            name="Countdown",
+            state_class=STATE_CLASS_MEASUREMENT,
```

Comment:
Is this a measurement or a setting?

Reply:
Absolutely correct this is a setting.

Reply:
Than it shouldn't be marked as measurement...

Reply:
Look at this again, countdown is actually a function as well, not just a status. Because of that reason, it should be a number entity, not a sensor.